assets/i18n/qqq.json

```diff
@@ -175,6 +175,18 @@
   "I18N_PLAYER_RATE_EXPLORATION": "Text displayed when the user is playing an exploration. - Text shown when the user finishes playing an exploration. On the bottom of the text, five stars appear for the user to give a rate to the exploration",
   "I18N_PLAYER_RATINGS_TOOLTIP": "Text displayed when the user is playing an exploration. - When the user clicks the information button, a dialog opens with further information. This text is shown in that dialog as tooltip of the exploration rating.",
   "I18N_PLAYER_RECOMMEND_EXPLORATIONS": "Text displayed when the user is playing an exploration. - Text shown above a list of recommended explorations to play next, after the user has finished playing the current exploration.",
+  "I18N_PLAYER_REPORT_MODAL_BODY_AD": "Text displayed when the user is reporting an exploration. - Text shown for ad option of report options in the flag exploraion modal.",
+  "I18N_PLAYER_REPORT_MODAL_BODY_ADDITIONAL_DETAILS": "Text displayed when the user is reporting an exploration. - Text shown for additional information textbox in the flag exploraion modal.",
+  "I18N_PLAYER_REPORT_MODAL_BODY_HEADER": "Text displayed when the user is reporting an exploration. - Text shown at the start of the body of the flag exploraion modal.",
+  "I18N_PLAYER_REPORT_MODAL_BODY_OTHER": "Text displayed when the user is reporting an exploration. - Text shown for other option of report options in the flag exploraion modal.",
```

Comment:
Clearer: Text shown for 'other' option in the 'flag exploration' modal.

Ditto below.